signup.html

```diff
@@ -0,0 +1,95 @@
+        </span>
+        
+        <div class='search-bar flex-horizontal-start flex-vertical-center'>
+            <span class='material-icons c-darkgray'>
+                search
+            </span>
+            <input type='text' class="padding-md no-outline no-border c-black" placeholder='Search' />
+        </div>
+        
+        <nav class='flex-vertical-center'>
+            <button class='btn-filled text-capital letter-space-md c-white no-border padding-md font-size-md margin-right-xxlg'>
+                login
+            </button>
+            <span class='material-icons card-icon c-white margin-right-xxlg'>
+                favorite_border
+            </span>
+            <span class='material-icons margin-right-xxlg card-icon c-white'>
+                shopping_cart
+            </span>
+        </nav>
+
+    </header>
+
+    <main class='flex-center'>
+        
+        <section class='flex-column padding-xxlg margin-top-xlg flex-center'>
+            <p class='text-capital font-size-lg letter-space-md margin-btm-md'>
+                signup
+            </p>
+            <form class='flex-column margin-horizontal-xxlg'>
+                <label for="email-input" class='text-capital margin-top-md c-black'>
+                    email address
+                </label>
+                <input type='email' id='email-input' class="padding-sm border-thin no-outline margin-top-sm" />
+                <label for="fname-input" class='text-capital margin-top-md c-black'>
+                    first name
+                </label>
+                <input type='text' id='fname-input' class="padding-sm border-thin no-outline margin-top-sm" />
+                <label for="lname-input" class='text-capital margin-top-md c-black'>
+                    last name
+                </label>
+                <input type='text' id='lname-input' class="padding-sm border-thin no-outline margin-top-sm" />
+                <label for='pass-input' class='text-capital margin-top-md c-black'>
+                    password
+                </label>
+                <input type='password' id='pass-input' class="'margin-top-sm no-outline padding-sm border-thin" />
+                <label for='cpass-input' class='text-capital margin-top-md c-black'>
+                    confirm password
+                </label>
+                <input type='password' id='cpass-input' class="'margin-top-sm no-outline padding-sm border-thin" />
+                <div class='flex-horizontal-end flex-vertical-center margin-top-md margin-btm-sm c-darkgray'>
+                    <input type='checkbox' id='toggle-pass' />
+                    <label for='toggle-pass' class='text-capital margin-left-sm'>
+                        show password
+                    </label>
+                </div>
+                <div class='flex-horizontal-start flex-vertical-center margin-top-sm margin-btm-lg c-darkgray'>
+                    <input type='radio' id='terms-check' />
```

Comment:
It is good to use a checkbox instead of a radio button here.

Reply:
radio replaced with checkbox